client/src/components/HeadManager.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Helmet } from 'react-helmet';
+import league_logo from '../assets/icons/league_logo.svg';
+
+const HeadManager = () => {
+  return (
+    <Helmet>
+      <title>Play Maker League</title>
```

Comment:
@keithj0nes I don't like hard coding things like this, we should create a resource string file